src/options.ts

```diff
@@ -65,6 +67,7 @@ export async function normalizeOptions(
     platform = 'node',
     outDir = 'dist',
     dts = false,
+    minify = true,
```

Comment:
I think it shouldn't be minified by default, just like tsup.

```suggestion
    minify = false,
```

Reply:
Thanks